ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/build_run_synthea.sh

```diff
@@ -304,23 +321,55 @@ download_props_file_from_s3(){
 prepare_and_run_synthea(){
   cd "${BFD_SYNTHEA_AUTO_LOCATION}"
   source .venv/bin/activate
-  python3 prepare-and-run-synthea.py "${BFD_END_STATE_PROPERTIES}" "${TARGET_SYNTHEA_DIR}" "${NUM_GENERATED_BENES}" "${UNIQUE_ENVS_PARAM}" "${SKIP_VALIDATION}"
+  python3 prepare-and-run-synthea.py "${BFD_END_STATE_PROPERTIES}" "${TARGET_SYNTHEA_DIR}" "${NUM_GENERATED_BENES}" "${UNIQUE_ENVS_PARAM}" "${NUM_FUTURE_MONTHS}" "${SKIP_VALIDATION}"
   deactivate
 }
 
+# Function that splits the synthea future lines into their own loads; only
+# does anything if future lines were generated.
+#
+#Args:
+# 1: location of synthea git repo
+#
+split_future_loads(){
+  if [ "${GENERATE_FUTURE}" == 'true' ]; then
+      cd "${BFD_SYNTHEA_AUTO_LOCATION}"
+      source .venv/bin/activate
+      python3 split-future-claims.py "${TARGET_SYNTHEA_DIR}"
+      deactivate
+  fi
+}
+
 # Function that invokes an S3 utility to upload new generated synthetic data to an
 # S3 folder where the BFD ETL pipeline will discover the synthetic RIF files and
 # load them into the appropriate BFD database.
 upload_synthea_results_to_s3(){
   
   echo "upload synthea results to S3"
   cd "${BFD_SYNTHEA_AUTO_LOCATION}"
+  
+  ## list out the future folders inside the output directory
+  FOLDERS=()
+  readarray -t FOLDERS < <(find "${BFD_SYNTHEA_OUTPUT_LOCATION}" -maxdepth 1 -mindepth 1 -type d -execdir basename {} ';')
+  echo "${#FOLDERS[@]} folders to upload found"
 
   # upload the RIF (.csv) files to S3 ETL bucket(s); once per environment, passed in as arg
   source .venv/bin/activate
   for s3_bucket in "${S3_BUCKETS[@]}"; do
     echo "uploading RIF files to: ${s3_bucket}"
     python3 ./s3_utilities.py "${BFD_SYNTHEA_OUTPUT_LOCATION}" "upload_synthea_results" "${s3_bucket}"
+    
+    if [ "$GENERATE_FUTURE" ]; then
```

Comment:
This is **always** _truthy_ with `GENERATE_FUTURE` defaulting to the string "false", e.g. `[ "false" ] && echo hi mom` and `[ "true" ] && echo hi mom` yield the same result. Borrowing from a similar conditional just above solves for this:
```suggestion
    if [ "$GENERATE_FUTURE" == 'true' ]; then
```

<details><summary>alternatively...</summary> 

You _could_ default `GENERATE_FUTURE` to an empty string, as `[ "" ]` is _falsey_, but I think the above is fine.

</details>

Reply:
Thanks, good catch! I guess i didn't catch this cause in the case there were no folders generated, it just did nothing anyway, but good to have it fixed to be correct.